pylib/anki/exporting.py

```diff
@@ -160,6 +160,12 @@ class AnkiExporter(Exporter):
     def __init__(self, col: _Collection) -> None:
         Exporter.__init__(self, col)
 
+    def deckIds(self) -> typing.List[int]:
```

Comment:
The typing. prefix is unnecessary, but otherwise looks good.

Reply:
Removed.
Some Typing remains in the code; such as typing.Union. A reason to keep them at some places and not other ?

Reply:
No reason to keep them - they were just added by a tool automatically, and nobody has gone through and manually removed the prefix and added the required import statement to the top.